platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -303,6 +303,7 @@ MGL_EXPORT
  */
 - (void)removePack:(MGLOfflinePack *)pack withCompletionHandler:(nullable MGLOfflinePackRemovalCompletionHandler)completion;
 
+- (void)invalidateOfflineRegionForPack:(MGLOfflinePack *)pack withCompletionHandler:(void (^)(NSError * _Nullable))completion;
```

Comment:
An “offline region” in mbgl jargon is equivalent to an “offline pack” in iOS/macOS-speak. In the public API, “offline region” usually refers to what’s known internally as an “offline region definition”. So `-invalidatePack:completionHandler:` would be less confusing.

Reply:
Thank you!